File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n=nums.length;
    let f_s=n*(n+1)/2;
    let sum=0;
    for (let i=0;i<nums.length;i++){
        sum=sum+nums[i];
    }
   
     let missing=f_s-sum;
  return missing;
    
};

File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        total = n * (n + 1) // 2
        count = 0

        for i in range(n):
            count += nums[i]

        return total - count

               